doc/api/process.markdown

```diff
@@ -286,6 +286,11 @@ event and that writes can block when output is redirected to a file (although
 disks are fast and operating systems normally employ write-back caching so it
 should be a very rare occurrence indeed.)
 
+Note that a callback function on `stream.write` might be called before all 
+data is flushed completly. The only way to ensure that all data to 
+`process.stderr` and `process.stdout` is written and flushed is to let 
+Node.js end itself.
+
```

Comment:
Hmm... perhaps, "Note that on `process.stdout` and `process.stderr`, the callback passed to `stream.write` might be called before all written data is flushed completely. This can result in data loss if Node.js is ended prematurely using `process.exit`."